.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: Repository (in owner/repo format)
+        required: true
+        default: kiali/kiali.io
+        type: string
+      releasing_branch:
+        description: Branch to release
+        required: true
+        default: staging
+        type: string
+
+jobs:
+  initialize:
+    name: Initialize
+    runs-on: ubuntu-20.04
+    outputs:
+      releasing_version: ${{ steps.releasing_version.outputs.releasing_version }}      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.releasing_branch }}      
+      
+      - name: Determine releasing version        
+        id: releasing_version
+        run: |
+          CURRENT_VERSION=$(awk '/## NEXT/,/url/' config.toml | grep -o '\"v.*\"' | tr -d '\"' | xargs)
+
+          # The following command removes the 'v' and adds the .z part of the version          
+          RAW_VERSION=$(echo "${CURRENT_VERSION:1}.0")
+          
+          echo "::set-output name=releasing_version::v$(./.github/workflows/util/semver bump minor $RAW_VERSION)"          
+      
+      - name: Log information
+        run: |
+          echo "Releasing version: ${{ steps.releasing_version.outputs.releasing_version }}"
+
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    needs: [initialize]
+    env:      
+      RELEASING_VERSION: ${{ needs.initialize.outputs.releasing_version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.releasing_branch }}    
+            
+      - name: Configure git
+        run: |
+          git config user.email 'kiali-dev@googlegroups.com'
+          
+          git config user.name 'kiali-bot'
+        
```

Comment:
Same comment as in the server side PR. We should find a replacement for this inactive e-mail.

Reply:
It is needed:

Author identity unknown

*** Please tell me who you are.

I searched a little bit but I only found workarounds for this.